docs/contributing/technical/resolve-merge-conflicts.md

```diff
@@ -11,6 +11,11 @@ Pretty often when opening a pull request it is very likely to run into merge con
 To better illustrate the commands listed here at will use commits and screenshots from [open-sauced#1078](https://github.com/open-sauced/open-sauced/pull/1078).
 
 
+## Repository setup
```

Comment:
@Lymah123 There is a duplication of this title in line 19. Which one do you want to keep?